census3/census3.go

```diff
@@ -0,0 +1,151 @@
+package census3
+
+import (
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/google/uuid"
+	c3types "github.com/vocdoni/census3/api"
+	c3client "github.com/vocdoni/census3/apiclient"
+)
+
+const (
+	// maxRetries is the maximum number of retries for a request.
+	maxRetries = 3
+	// cooldown is the time to wait between retries.
+	cooldown = time.Second * 2
+)
+
+// Client wraps a client for the Census3 API and other revelant data.
+type Client struct {
```

Comment:
tbh I'm not sure for the need of this wrapper of the census3 client (which is already a wrapper for the API).
The retry mechanism could also be implemented on the original client instead of here.

What's the point @jordipainan ?

Reply:
I like to wrap these kind of imports so they can be extended on the consumer, imo adds more flexibility i.e: a new attribute needs to  be added.

I dont think the retry mechanism should be added on the base client, and I think thats up to the consumer too. The base client needs to be simplest as possible and the retry mechanism is something one may decide to add or not.

Reply:
Most of the methods for the wrapper are just calls to the census3 client with the retry helper. I still don't see the reason to have this wrapper, tbh. 

Adding a wrapper adds complexity, and I'm not sure about the flexibility thing. Keep it simple if possible. 

Anyway, let's call for more feedback @lucasmenendez